### ISSUE TITLE
[macOS] Dota 2 will not launch on macOS 10.13.6

### ISSUE THREAD

**Issue by tabanger** [ValveSoftware/Dota-2#2791]
#### Your system information

* System information from steam:  https://gist.github.com/tabanger/9aab5973041bd085173361607ca11114
* Have you checked for system updates?: Yes
* Are you using the latest stable video driver available for your system? Yes
* Have you verified the game files?: No

#### Please describe your issue in as much detail as possible:
Since the Ringmaster update, Dota 2 will no longer launch on my computer, running macOS 10.13.6, at all.  It worked fine yesterday, before the update.  According to the the crashlog, it appears that the dota2 binary has been built for macOS 10.15 as the minimum.  Crashlog:

````
Process:               dota2 [34462]
Path:                  /Users/USER/Library/Application Support/Steam/*/dota2.app/Contents/MacOS/dota2
Identifier:            dota2
Version:               1.0.0
Code Type:             X86-64 (Native)
Parent Process:        bash [34457]
Responsible:           dota2 [34462]
User ID:               501

Date/Time:             2024-08-23 13:39:55.639 -1000
OS Version:            Mac OS X 10.13.6 (17G14042)
Report Version:        12
Anonymous UUID:        25617244-EFEA-657F-FC0D-09D267A630B0


Time Awake Since Boot: 11000000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  MainThrd  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x4] Symbol missing

Dyld Error Message:
  Symbol not found: ____chkstk_darwin
  Referenced from: /Users/USER/Library/Application Support/Steam/*/dota2.app/Contents/MacOS/dota2 (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib
````

Was this a build mistake?  I could not find any release notes or announcements that indicated that the minimum OS requirements had been raised.  The Store page still lists macOS 10.13.6 as the official minimum OS required:
https://store.steampowered.com/app/570/Dota_2/

#### Steps for reproducing this issue:

1.  Have latest dota2 update installed
2. Be running macOS 10.13 or 10.14
3. Game crashes immediately upon trying to launch.

**Comment by tje3d** [comment]
Running the Dota 2 binary directly, instead of through Steam, will fix the problem, and you won't experience crashes.

**Comment by tabanger** [comment]
> Running the Dota 2 binary directly, instead of through Steam, will fix the problem, and you won't experience crashes.

That had no effect.  Launching the binary directly led to the same failure to launch, with the crash report containing the same infomation as above.

**Comment by NikMaslukovFido** [comment]
Same issue on mac os 13.2, can't launch dota via steam/binary

**Comment by mehdinourollah** [comment]
Same here on 15.0 Beta (24A5327a)...Tried even running the binary but no difference

**Comment by mehdinourollah** [comment]
I found this fix from Steam forums (I'm on M3 Pro Sequoia)
https://steamcommunity.com/app/570/discussions/0/4432191936062478720/?ctp=2#c4432191936062803170

**Comment by 3okash** [comment]
Run this command from terminal and it should work fine.
`open -n ~/Library/Application\ Support/Steam/steamapps/common/dota\ 2\ beta/game/bin/osx64/dota2.app --args -map dota -novid -vulkan -nojoy -novr -high -prewarm`

**Comment by tabanger** [comment]
> Same issue on mac os 13.2, can't launch dota via steam/binary

> Same here on 15.0 Beta (24A5327a)...Tried even running the binary but no difference

> I found this fix from Steam forums (I'm on M3 Pro Sequoia) https://steamcommunity.com/app/570/discussions/0/4432191936062478720/?ctp=2#c4432191936062803170

> Run this command from terminal and it should work fine. `open -n ~/Library/Application\ Support/Steam/steamapps/common/dota\ 2\ beta/game/bin/osx64/dota2.app --args -map dota -novid -vulkan -nojoy -novr -high -prewarm`

No, none of that works or has any effect.  **Please stop posting in this bug if you have Apple Silicon.**  This bug is only for running on macOS 10.13 and 10.14.  This has nothing to do with Apple Silicon or Error 24.

**Comment by tabanger** [comment]
More corroborating reports on 10.13 and 10.14 here:

https://steamcommunity.com/app/570/discussions/0/4768836464219227720/
https://steamcommunity.com/app/570/discussions/0/3825299364570303003/?ctp=3#c3825299364570998870

**Comment by DaniiS97** [comment]
any updates?

**Comment by jzuleta7** [comment]
> Run this command from terminal and it should work fine. `open -n ~/Library/Application\ Support/Steam/steamapps/common/dota\ 2\ beta/game/bin/osx64/dota2.app --args -map dota -novid -vulkan -nojoy -novr -high -prewarm`

Thank you! This worked for me on MacOs Ventura 13.70 

As a suggestion after the game loads change the resolution to windowed  and something lower than your current desktop.

**Comment by andrew-zzz** [comment]
hey gays,i also meet this problem with m1，please update the latest mac system to 15.6 can fix it
